features/outbox.feature

```diff
@@ -9,6 +9,19 @@ Feature: Outbox
     And a "Announce(C)" Activity "An" by "Alice"
     And a "Follow(Us)" Activity "F" by "Alice"
     And a "Accept(F)" Activity "A" by "Alice"
+    And "Alice" adds "C" to the Outbox
+    And "Alice" adds "An" to the Outbox
+    And "Alice" adds "F" to the Outbox
+    And "Alice" adds "A" to the Outbox
     When the contents of the outbox is requested
-    Then the outbox contains 1 activity
-    And a "Create(Article)" activity is in the Outbox
+    Then the outbox contains 2 activities
```

Comment:
Realise I goofed up this test first time round and it wasn't doing what it should have been 🙃

Reply:
Changed my mind about this and switched to covering this with a unit test